cmd/subcommands/staking.go

```diff
@@ -71,7 +71,7 @@ var (
 )
 
 func getNextNonce(addr oneAddress, messenger rpc.T, nonce int64) uint64 {
-	if nonce < 0 {
+	if nonce != -1 {
 		return uint64(nonce)
```

Comment:
So this returns what on negative input ? For -3, gives back 3 ?

Reply:
Changed, check latest